pkg/chaosdaemon/server.go

```diff
@@ -45,22 +46,26 @@ func newServer() (*Server, error) {
 	}, nil
 }
 
-func StartServer(host string, port int) {
+// StartServer starts chaos-daemon.
+func StartServer(host string, port int, containerRuntime string) error {
 	lis, err := net.Listen("tcp", fmt.Sprintf("%s:%d", host, port))
 	if err != nil {
-		log.Error(err, "failed to listen")
+		return errors.Wrap(err, "failed to listen")
```

Comment:
Don't use `errors.Wrap` in this pr first, you can push a single PR later to unify all error handling.

Reply:
Do you mean that I just need to log and return that error directly?

Reply:
Done